plot/PlotInvariantMassFits.C

```diff
@@ -0,0 +1,271 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
```

Comment:
Same comment as above (plus this is a duplication if I got properly): I would not include this macro in the repository (we typically keep outside or in other repositories the macros for plots, otherwise they proliferate too much)